### PR TITLE
Player, Telemetry and api changes

### PR DESCRIPTION
**Pull request by Aar-if**
Player web-component update 
Tracking API changes
Telemetry changes
Duration added
Program ID added
Content - type added

### CODE REVIEW

File: packages/common-lib/src/components/SunbirdPlayer.js [hunk 1]
```diff
@@ -12,6 +12,12 @@ const SunbirdPlayer = ({
   ...props
 }) => {
   const { mimeType } = props
+
+  const typeMatch = mimeType.match(/\/(.+)$/)
+  const fileType = typeMatch ? typeMatch[1] : null
+
+  localStorage.setItem('contentType', fileType)
```
Comment (snehal0904) [inline packages/common-lib/src/components/SunbirdPlayer.js:19]: @Aar-if Can we clear local storage once the use of this variable done?

Comment (Aar-if) [reply packages/common-lib/src/components/SunbirdPlayer.js:19]: yes

File: packages/common-lib/src/services/subjectListRegistryService.js [hunk 2]
```diff
@@ -21,6 +21,8 @@ export const getProgramId = async (props) => {
     { headers }
   )
   if (programID?.data?.data) {
+    console.log(programID?.data?.data[0]?.programId)
```
Comment (snehal0904) [inline packages/common-lib/src/services/subjectListRegistryService.js:24]: @Aar-if console remove
Program Id we can keep as it is in local storage as it is same till the session close & this year

File: packages/players/quml/assets/styles/quml-carousel.css [hunk 3]
```diff
@@ -0,0 +1,240 @@
+html, body {
+    height: 100%;
+  }
+  .carousel {
```
Comment (snehal0904) [inline packages/players/quml/assets/styles/quml-carousel.css:4]: @Aar-if This is not EG specific na?

File: packages/players/quml/carousel.css
Comment (snehal0904) [inline packages/players/quml/carousel.css:1]: @Aar-if is this empty file required

File: packages/players/quml/assets/styles/quml-carousel.css (hunk 3 above)
Comment (Aar-if) [reply packages/players/quml/assets/styles/quml-carousel.css:4]: no

File: packages/players/quml/carousel.css
Comment (Aar-if) [reply packages/players/quml/carousel.css:1]: checking

File: packages/players/quml/index.html [hunk 4]
```diff
@@ -44,10 +46,15 @@
...
       window.questionListUrl = "https://vdn.diksha.gov.in/api/question/v1/list";
+      if (localStorage.getItem("console")) {
+        console.log("window?.name", window?.name);
+      }
       if (window?.name) {
         const data = JSON.parse(window?.name);
         if (data?.questionListUrl) {
-          console.log("asd", data?.questionListUrl);
+          if (localStorage.getItem("console")) {
+            console.log("asd", data?.questionListUrl);
```
Comment (snehal0904) [inline packages/players/quml/index.html:56]: @Aar-if This is alos looks testing, remove

File: packages/players/quml/index.html [hunk 5]
```diff
@@ -63,7 +70,9 @@
       // });
 
       qumlPlayerElement.addEventListener("telemetryEvent", (event) => {
-        console.log("On telemetryEvent", event);
+        if (localStorage.getItem("console")) {
```
Comment (snehal0904) [inline packages/players/quml/index.html:73]: @Aar-if This also

File: packages/players/quml/styles.css [hunk 6]
```diff
@@ -4880,7 +4435,7 @@ html[dir="rtl"] .sb-left-icon-btn i.icon {
 .ui.toggle.checkbox.sb-toggle input:focus:checked ~ .box:before,
 .ui.toggle.checkbox.sb-toggle input:checked ~ .box:before,
 .ui.toggle.checkbox.sb-toggle input:checked ~ label:before {
-  background-color: #024f9d !important;
+  background-color: red !important;
```
Comment (snehal0904) [inline packages/players/quml/styles.css:4438]: @Aar-if Looks like EG specific change. Please remove

Comment (Aar-if) [reply packages/players/quml/styles.css:4438]: yes

Comment (Aar-if) [reply packages/players/quml/styles.css:4438]: part of the video player itself not EG specific

**Pull request merged**